op-service/sources/l1_beacon_client.go

```diff
@@ -129,6 +129,19 @@ func (cl *BeaconHTTPClient) BeaconBlobSideCars(ctx context.Context, fetchAllSide
 	if err := cl.apiReq(ctx, &resp, reqPath, reqQuery); err != nil {
 		return eth.APIGetBlobSidecarsResponse{}, err
 	}
+
+	indices := make(map[uint64]bool)
```

Comment:
```suggestion
	indices := make(map[uint64]struct{}, len(hashes))
```
better pre-allocation, and we can work with an empty struct here